package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-authenticate-me",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
We will have to support two version of this gem, 1) Ember 1.x, and 2) Ember 2.x.
Can we bump this to "0.6.0", to leave some space and identify this as a big change?